lib/hawkular/hawkular_client.rb

```diff
@@ -66,7 +66,7 @@ def tokens
 
     # this is in a dedicated method, because constructor opens the websocket connection to make the handshake
     def init_operations_client
-      Operations::OperationsClient.new(entrypoint: @state[:entrypoint],
+      Operations::Client.new(entrypoint: @state[:entrypoint],
```

Comment:
It *could* but not sure if anyone is doing it. 
Anyway i'll use a #create method.